Automattic/wp-calypso
# Editor: Update classic block guide svg

#### Changes proposed in this Pull Request

* Replace classic block svg with one that has internal padding

#### Testing instructions

- Set a user attribute for your user using wpsh on a sandbox $ update_user_attribute( USER_ID, 'editor_deprecation_group', true )
- Check out this PR and build the wpcom block editor files with yarn build:prod in the apps/wpcom-block-editor/package.json folder.
- Sync the files created by the build in the '/dist' folder with your sandbox widgets/wpcom-block-editor folder
- Also apply D45786-code to sandbox
- Make sure site used above is sandboxed Load editor editor again and make sure the Classic block guide loads instead of the standard WPcom guide

Before:
<img width="278" alt="Screen Shot 2020-07-14 at 9 29 08 AM" src="https://user-images.githubusercontent.com/3629020/87355704-eda4b100-c5b4-11ea-9df7-694b44e2db63.png">

After:
<img width="285" alt="Screen Shot 2020-07-14 at 10 59 34 AM" src="https://user-images.githubusercontent.com/3629020/87361941-745f8b00-c5c1-11ea-9284-7588fc3f42cf.png">
